### ISSUE TITLE
USERNAME placeholder

### ISSUE THREAD

**Issue by Shockingg** [VertisanPRO/pterodactyl#201]
### Product

WemX Pro

### Describe the issue

I don't think the USERNAME checkout variable works, I've tried it many times and come to the conclusion that it must be some kind of bug as the other variables work fine. I'm not 100% sure as no one responds in the discord lmao, so it might just be an issue I'm having. Whenever i create a server it just says USERNAME, and does not work as intended, but the other placeholder work great.

### Pterodactyl Panel Version

1.11.3

### Product Version

1.5.0

**Comment by Shockingg** [comment]
> Its been a known bug since 1.3.x I think @GIGABAIT93 gave up or forgot this issue. I also believe random text / numbers don't work. I know AUTO_PORT works as intended.

Yes I know both the AUTO_PORT and RANDOM_TEXT variables work fine, I haven't tested the others tho. But thanks for confirming the USERNAME variable is still a bug.

**Comment by VanillaChan6571** [comment]
> > Its been a known bug since 1.3.x I think @GIGABAIT93 gave up or forgot this issue. I also believe random text / numbers don't work. I know AUTO_PORT works as intended.
> 
> Yes I know both the AUTO_PORT and RANDOM_TEXT variables work fine, I haven't tested the others tho. But thanks for confirming the USERNAME variable is still a bug.

Ah okay, Thanks for letting me know.

**Comment by GIGABAIT93** [comment]
Hi what error exactly are you getting?

**Comment by GIGABAIT93** [comment]
Oh I found the problem will be fixed in the next update
